### PR TITLE
24 - Initial implementation of progress bar

### LINKED ISSUE #23: Add Progress Bar
As a developer, I would like to see program progress (% complete or indicator that the application is still running) from command line console.

### PR DESCRIPTION
**Pull request by jigglepuff**
Closes: #23 

**Changes**:
- Moved **constant values** from `app.py` to `etl/constants.py`
  - Deleted/moved empty top-level `constants.py` file
- Replaced print statements with **logs**
  - Package: [logging](https://docs.python.org/3/library/logging.html#module-logging)
  - Added logger config file: `data/logger/config.ini`
  - Added logger initialization in __init__ of all classes
- Added **progress bar**
  - Package: [enlighten](https://python-enlighten.readthedocs.io/en/stable/api.html#module-enlighten)
  - Created `ProgressBarManager` singleton class (see `progress_bar_manager.py`)
  - Added logic to estimate # of jobs
  - Added calls to update progress bar
- Refactor **Parser**
  - Created `parse_all()` function to mirror call pattern of Fetcher and Transformer
- Refactor **Extractor**
  - Created `extract_all()` function to mirror call pattern of Fetcher and Transformer
- Refactor **Loader**
  - Created `load_all()` function to mirror call pattern of Fetcher and Transformer
- Update `sources.yml`
  - Changed names of keys to be more descriptive to the dataset
  - Added info field for dataset description URL
- Update `requirements.txt`
  - Added new package dependencies

### CODE REVIEW

File: data/sources/sources.yml [hunk 1]
```diff
@@ -1,20 +1,27 @@
...
+  info: https://www.stlouis-mo.gov/data/datasets/distribution.cfm?id=83
   url: https://www.stlouis-mo.gov/data/upload/data-files/prcl.zip
 
-par:
+Parcel_Data:
+  info: https://www.stlouis-mo.gov/data/datasets/distribution.cfm?id=85
   url: https://www.stlouis-mo.gov/data/upload/data-files/par.zip
 
-lra_public:
+LRA_Iventory_Records:
```
Comment (mrpetrocket) [inline data/sources/sources.yml]: LRA_Inventory_Records

File: etl/constants.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
+# Declare global variables (callable across files)
+TOTAL_STAGES = 5
```
Comment (mrpetrocket) [inline etl/constants.py]: What is `TOTAL_STAGES` for?

**Review by mrpetrocket — APPROVED** [review]
I love the look of the progress bars.

File: etl/constants.py (hunk 2 above)
Comment (jigglepuff) [reply etl/constants.py]: It was originally for a master progress bar (which will track child progress bars for each stage). I didn't end up implementing that, so I deleted the unused constant for now.

File: data/sources/sources.yml (hunk 1 above)
Comment (jigglepuff) [reply data/sources/sources.yml]: Fixed typo.

**Pull request merged**